apis/identity/v1alpha1/zz_oidcclient_types.go

```diff
@@ -67,6 +67,7 @@ type OidcClientObservation struct {
 	// A list of assignment resources associated with the client.
 	Assignments []*string `json:"assignments,omitempty" tf:"assignments,omitempty"`
 
+	// The Client ID returned by Vault.
 	// The Client ID from Vault.
```

Comment:
Nitpick, I would clean up the duplicate comment here.

Reply:
This is auto-generated code we don't have much of a choice.

Reply:
There's an open issue for this bug upjet: https://github.com/crossplane/upjet/issues/371